toolkit/tools/grapher/grapher.go

```diff
@@ -25,6 +25,18 @@ var (
 	strictGoals      = app.Flag("strict-goals", "Don't allow missing goal packages").Bool()
 	strictUnresolved = app.Flag("strict-unresolved", "Don't allow missing unresolved packages").Bool()
 
+	resolveCylesFromUpstream = app.Flag("resolve-cycles-from-upstream", "Let grapher resolve cycles by marking rpms available in repo as remote").Bool()
+	outDir      = exe.OutputDirFlag(app, "Directory to download packages into.")
+        existingRpmsDir          = app.Flag("rpm-dir", "Directory that contains already built RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
+        existingToolchainRpmDir = app.Flag("toolchain-rpms-dir", "Directory that contains already built toolchain RPMs. Should contain top level directories for architecture.").Required().ExistingDir()
+        tmpDir                  = app.Flag("tmp-dir", "Directory to store temporary files while downloading.").String()
+
+        workerTar            = app.Flag("tdnf-worker", "Full path to worker_chroot.tar.gz").Required().ExistingFile()
+        repoFiles            = app.Flag("repo-file", "Full path to a repo file").Required().ExistingFiles()
+        usePreviewRepo       = app.Flag("use-preview-repo", "Pull packages from the upstream preview repo").Bool()
+        toolchainManifest    = app.Flag("toolchain-manifest", "Path to a list of RPMs which are created by the toolchain. Will mark RPMs from this list as prebuilt.").ExistingFile()
+	ignoreVersionToResolveSelfDep = app.Flag("ignore-version-to-resolve-selfdep", "Ignore package version while downloading package from usptream when resolving cycle").Bool()
```

Comment:
Once we're set on the flags, could you fix the indents? In general, we use tabs everywhere in Go code. If you're using VSCode, then the `golang.go` extension should be able to fix these for you automatically.

Reply:
fixed indentation.

Reply:
Thanks, looks better! Could you also align the `=` parts?

Reply:
There are a few ways to auto format the code as well.
- As @PawelWMS said vscode's go extension is a great one
- `cd toolkit && make go-fmt-all`
- `cd toolkit/tools  && go fmt ./...` (yes, that is meant to be 3 '.'s)